
2022-10-25 01:10:11.537 - [Debug] [Auth] Attacker 120.236.79.182 trying to authenticate with 'none'
2022-10-25 01:10:11.538 - [Debug] [Auth] Attacker authentication error: No authentication method provided
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
(node:2288323) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_4',
  containerIp: '10.0.3.127',
  mitmPort: 1003,
  autoAccessFixed: 1
}
2022-10-25 01:11:16.548 - [Info] MITM Version: 2
2022-10-25 01:11:16.549 - [Info] Auto Access Enabled: true
2022-10-25 01:11:16.549 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-25 01:11:16.556 - [Info] SSH man-in-the-middle server for 10.0.3.127 listening on 10.0.3.1:1003
2022-10-25 01:11:19.021 - [Debug] [Connection] Attacker connected: 223.171.91.177 | Client Identification: SSH-2.0-Go
2022-10-25 01:11:19.140 - [Debug] [Connection] Attacker connected: 180.93.32.58 | Client Identification: SSH-2.0-Go
2022-10-25 01:11:19.187 - [Debug] [Connection] Attacker connected: 102.141.225.238 | Client Identification: SSH-2.0-Go
2022-10-25 01:11:19.789 - [Debug] [Connection] Attacker connected: 174.4.208.151 | Client Identification: SSH-2.0-Go
2022-10-25 01:11:20.430 - [Debug] [Auth] Attacker 174.4.208.151 trying to authenticate with 'none'
2022-10-25 01:11:20.431 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 01:11:20.519 - [Debug] [Auth] Attacker 223.171.91.177 trying to authenticate with 'none'
2022-10-25 01:11:20.519 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 01:11:20.623 - [Debug] [Auth] Attacker 174.4.208.151 trying to authenticate with 'publickey'
2022-10-25 01:11:20.625 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-25 01:11:20.625 - [Debug] [Auth] Attacker: 174.4.208.151 has so far made 1 attempts. Remaining: 5 attempts
2022-10-25 01:11:20.704 - [Debug] [Auth] Attacker 180.93.32.58 trying to authenticate with 'none'
2022-10-25 01:11:20.704 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 01:11:20.710 - [Debug] [Auth] Attacker 102.141.225.238 trying to authenticate with 'none'
2022-10-25 01:11:20.710 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 01:11:20.827 - [Debug] [Connection] Attacker closed the connection
2022-10-25 01:11:20.929 - [Debug] [Connection] Attacker connected: 174.4.208.151 | Client Identification: SSH-2.0-Go
2022-10-25 01:11:21.039 - [Debug] [Auth] Attacker 223.171.91.177 trying to authenticate with 'publickey'
2022-10-25 01:11:21.039 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-25 01:11:21.040 - [Debug] [Auth] Attacker: 223.171.91.177 has so far made 1 attempts. Remaining: 5 attempts
2022-10-25 01:11:21.205 - [Debug] [Auth] Attacker 102.141.225.238 trying to authenticate with 'password'
2022-10-25 01:11:21.205 - [Debug] [Auto Access] Attacker: 102.141.225.238, Threshold: 1, Attempts: 1
2022-10-25 01:11:21.205 - [Debug] [Auto Access] Compromising the honeypot
2022-10-25 01:11:21.205 - [Debug] [Auto Access] Adding the following credentials: 'root:1'
2022-10-25 01:11:21.270 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2022-10-25 01:11:21.272 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-25 01:11:21.273 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.127
2022-10-25 01:11:21.290 - [Debug] [Auth] Attacker 180.93.32.58 trying to authenticate with 'publickey'
2022-10-25 01:11:21.295 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-25 01:11:21.296 - [Debug] [Auth] Attacker: 180.93.32.58 has so far made 1 attempts. Remaining: 5 attempts
